Owl/app/src/main/java/com/materialstudies/owl/ui/lessons/LessonBottomSheetBehavior.kt

```diff
@@ -0,0 +1,19 @@
+package com.materialstudies.owl.ui.lessons
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+
```

Comment:
It might be helpful to add a brief comment as to why this behavior subclass is necessary/what problem it solves.

Reply:
sure, i've added a comment for this behavior